### PR TITLE
feat: add Footprint object which can calculate the compensation price

### PR DESCRIPTION
**Pull request by Rocksheep**

### CODE REVIEW

**Comment by JaZo** [comment]
:information_source: Now with type hints in your code, defining types within PHP DocBlocks is redundant. Laravel has removed all of the `@param` and `@return` tags from its DocBlocks where types are defined with PHP. Similarly, Shift [removed these tags](https://github.com/swisnl/dlf-hackathon-2023/pull/23/commits/525ea37145d879a3d06d9e4e30a5b5cde0f526d2?w=1) from any DocBlock where the code now has _equivalent_ type hints.

**Comment by JaZo** [comment]
:information_source: While the _Laravel Fixer_ performs automation to adopt the latest Laravel conventions, Shift understands developers have different preferences. All of Shift's automation is done in nice, atomic commits. This makes it easier to undo any of the changes Shift makes using `git revert`.

File: app/Services/CO2viaCnaught.php [hunk 1]
```diff
@@ -11,12 +12,12 @@ public function __construct(
...
 
-    public function getCO2InGrams(float $distanceInKm, float $massKg = 0): float
+    public function getCO2Footprint(float $distanceInKm, float $massKg = 0): CO2Footprint
     {
         $response = Http::withToken($this->apiKey)
             ->post('https://api.cnaught.com/v1/quotes/ride', ['distance_km' => $distanceInKm])
             ->json();
 
-        return $response['amount_kg'] * 1000;
+        return new CO2Footprint($response['amount_kg'] / 1000);
```
Comment (JaZo) [inline app/Services/CO2viaCnaught.php]: Shouldn't this be a multiplier instead of division?

**Review by JaZo — CHANGES_REQUESTED** [review]

File: app/Services/CO2viaCnaught.php (hunk 1 above)
Comment (Rocksheep) [reply app/Services/CO2viaCnaught.php]: You're right. I will delete the entire file.

**Review by JaZo — CHANGES_REQUESTED** [review]
Please also remove the config and example env.

**Review by JaZo — APPROVED** [review]

**Pull request merged**